src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -174,15 +173,18 @@ private static void addConnectorConfigs(final ConfigDef configDef) {
         configDef.define(
                 CONNECTION_URL_CONFIG,
                 Type.LIST,
-                new ArrayList<>(),
```

Comment:
Is this reverting `https://github.com/aiven/opensearch-connector-for-apache-kafka/commit/298abbb222331472c690b83017c950a14a2a7bdf` ?

Reply:
This specific change - yes but otherwise - not completely, the test from previous PR is still there but slightly modified.